### PR TITLE
Seeding countries and breads

### PR DESCRIPTION
**Pull request by EricBelongea**
## What kind of change does this PR introduce?
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ] Small fix (Corrected a typo, removed a comment, etc.)

## Checklist before requesting a review
- [ ] I ran the code locally and verified that there are no visible errors.
- [ ] I have performed a self-review of my code
- [ ] I have refactored my code before this push
- [ ] The commit message follows our guidelines
- [ ] My code follows the code style of this project.

### CODE REVIEW

**Review by brendan-bondurant — COMMENTED** [review]
That's a lot of bread

File: app/controllers/api/v1/users_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def show
     user = User.find(params[:id])
     render json: UserSerializer.new(user), status: 200
   end
-
+# POST /api/v1/users
```
Comment (coryrpow) [inline app/controllers/api/v1/users_controller.rb:12]: Added this in for FE and talked through it with them so they could see a little better what was going on.

**Review by coryrpow — COMMENTED** [review]
Added in all the breads with images in the seeds in addition to the countries that Eric added in. Also added a comment in the user controller to help the FE better see what's doing what.

File: app/controllers/api/v1/users_controller.rb (hunk 1 above)
Comment (brendan-bondurant) [reply app/controllers/api/v1/users_controller.rb:12]: Cool, that looks helpful

**Pull request merged**